app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,13 @@
+<resources>
+    <string name="app_name">Trippe</string>
+    <string name="title_home">Trips</string>
+    <string name="title_dashboard">Calendar</string>
+    <string name="title_notifications">Currency</string>
+    <string-array name="currency_array">
+        <item>-</item>
```

Comment:
why are you adding a "-" at the top of the list? I would personally just take this out and have a default to fall to.

Reply:
done